apps/teamsjs-diagnostic-app/src/components/sample/LogConsole.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import './LogConsole.css';
+
+interface LogConsoleProps {
+  initialLogs?: string[];
+}
+
+const MAX_LOGS = 100;
+
+const LogConsole: React.FC<LogConsoleProps> = ({ initialLogs = [] }) => {
+  const [logStatements, setLogStatements] = useState<string[]>(initialLogs);
+
+  const captureConsoleLogs = (...args: any[]) => {
```

Comment:
I would still encourage you to use logger something like Trevor mentioned https://www.npmjs.com/package/debug

Reply:
I will focus on trying to implement this in my MVP! I think for now it is a P2 priority!